src/components/common/ExpressionInput.vue

```diff
@@ -17,7 +17,9 @@
         :class="['form-control', {'is-invalid': isInvalid}]"
         :rows="rows"
         :placeholder="placeholder"
-        @input="$emit('input', $event.target.value)" />
+        @click="debounce_portAutoSuggestContent()"
```

Comment:
this is required in case the user re-click at the input without updating the expression